package.json

```diff
@@ -0,0 +1,24 @@
+{
```

Comment:
This is an NPM package file. Amongst other stuff, it specifies the dependencies of our package.